kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/VirtualKafkaClusterReconciler.java

```diff
@@ -297,6 +290,48 @@ public List<EventSource<?, VirtualKafkaCluster>> prepareEventSources(EventSource
                 new InformerEventSource<>(clusterToKubeService, context));
     }
 
+    @VisibleForTesting
+    static SecondaryToPrimaryMapper<KafkaProtocolFilter> filterSecondaryToPrimaryMapper(EventSourceContext<VirtualKafkaCluster> context) {
+        return filter -> {
+            if (!ResourcesUtil.isStatusFresh(filter)) {
+                LOGGER.debug("Ignoring event from KafkaProtocolFilter with stale status: {}", ResourcesUtil.toLocalRef(filter));
```

Comment:
brace with isDebugEnable, or use the fluent API to avoid the `ResourcesUtil.toLocalRef` call if DEBUG is off.